lightstep/util.py

```diff
@@ -20,9 +20,7 @@ def _generate_guid():
     """
     Construct a guid - random 64 bit integer converted to a string.
     """
-    # Note: uuid.uuid4() returns 128 bit int. To get 64 bit int, apply the mask.
-    guid = uuid.uuid4().int & (1<<64)-1
-    return str(guid)
+    return str(random.getrandbits(64))
```

Comment:
My two cents: let's avoid affecting the seeding the *global* rng. Let's create our own instance of a RNG if it's not seeded by default. I've been burned by trying to set specific seed on the global RNG only to find out a third-party library I was using re-seeding it to something random.